frontend_tests/node_tests/stream_data.js

```diff
@@ -438,6 +438,12 @@ zrequire('marked', 'third/marked/lib/marked');
     assert(!stream_data.is_subscribed('Canada'));
     assert(!stream_data.get_sub('Canada'));
     assert(!stream_data.get_sub_by_id(canada.stream_id));
+
+    blueslip.warn = function (msg) {
+        assert.equal(msg, 'Failed to delete stream EEXIST');
```

Comment:
Why "EEXIST" instead of "EXIST"?

Reply:
Ah, i failed to delete it since the stream does not exists.

Reply:
Ah, it failed to delete it since the stream does not exists.

Reply:
changing it to `does_not_ exists` to be more clear.